docs/user-guide/walkthrough/README.md

```diff
@@ -134,7 +134,7 @@ For this example, we'll be creating a Redis pod, with a named volume and volume
       emptyDir: {}
   ```
 
-1. Define a volume mount within a container definition:
+2. Define a volume mount within a container definition:
```

Comment:
Ah, I see that github actually handles this correctly (second item is 2), but the generated HTML on kubernetes.io does not (http://kubernetes.io/v1.0/docs/user-guide/walkthrough/k8s201.html). I don't know whether this will fix the generated HTML, or if it will be ignored and stay 1